website_sale_product_pack/controllers/variant.py

```diff
@@ -0,0 +1,31 @@
+from odoo.http import request, route
+
+from odoo.addons.website_sale.controllers.variant import WebsiteSaleVariantController
+
+
+class WebsiteSaleVariantController(WebsiteSaleVariantController):
+    @route(
+        "/website_sale/get_combination_info",
+        type="json",
+        auth="public",
+        methods=["POST"],
+        website=True,
+    )
```

Comment:
Why are you rewriting the route property? In the previous controller you didn't do it.

Reply:
Good point, I removed that